Tests/SwiftSyntaxBuilderTest/ImportTests.swift

```diff
@@ -7,9 +7,12 @@ final class ImportTests: XCTestCase {
     let leadingTrivia = Trivia.unexpectedText("␣")
     let identifier = TokenSyntax.identifier("SwiftSyntax")
 
-    let importDecl = ImportDecl(path: AccessPath([AccessPathComponent(name: identifier)]))
+    let importDecl = ImportDecl(
+      leadingTrivia: leadingTrivia,
```

Comment:
Actually, I don’t know why we always add the `␣` as leading trivia here. I think it’s something that was done in one of the first test cases and then just carried over to all the other tests. I would be totally fine to remove `␣` from all test cases because it’s rather weird anyway. I’m fine to keep it as-is in this PR though.